packages/datadog-plugin-cucumber/test/index.spec.js

```diff
@@ -54,6 +56,9 @@ describe('Plugin', function () {
   let Cucumber
   this.timeout(10000)
   withVersions('cucumber', '@cucumber/cucumber', version => {
+    const specificVersion = require(`../../../versions/@cucumber/cucumber@${version}`).version()
+    if ((NODE_MAJOR === 14 || NODE_MAJOR === 16) && semver.satisfies(specificVersion, '>=10')) return
```

Comment:
Why not `NODE_MAJOR<16` ?

Reply:
of course 🤦 . I was thinking these were strings and I was doing `=== '16'`. Good catch